io-pay-portal-fe/src/index.pug

```diff
@@ -6,7 +6,7 @@ html(lang='it')
         meta(robots="noindex, nofollow")
         link(href="scss/style.scss",rel="stylesheet")
         if process.env.IO_PAY_PORTAL_ENV=='production'
```

Comment:
maybe the environment is prod and not production,@MarcoAGreco,  are you verifying it in another pr ?

Reply:
we can merge this PR after --> https://github.com/pagopa/io-pay-portal/pull/118/files